bin/vmcloak-iptables

```diff
@@ -1,9 +1,28 @@
-#!/bin/sh
-
-# Credits to Mark Schloesser, https://github.com/rep/cuckoo-contrib
+#!/bin/bash
+
+if [ $# -ne 0 ]; then
+   if [[ $1 == '-h' ]]; then
+       echo "Usage:"
+       echo "    $0 ip_range/CIDR"
+       echo "    $0 ip_range/CIDR interfaces"
+       echo "    $0 192.168.56.0/24 eth0 wlan0"
+       echo ""
+       echo "By default will be:"
+       echo "    $0 192.168.56.0/24 eth0 wlan0"
+       echo ""
```

Comment:
Yeah, same here, but I'd also like it to work on my macbook while we're at it ;)
If you just do another push to this pull request you can include the ``exit 0``, or otherwise I can do it for you (after I test the stuff).